pkg/platform/api/platform.go

```diff
@@ -0,0 +1,142 @@
+package api
+
+import (
+	"github.com/Masterminds/semver"
+	"github.com/bottlerocket-os/bottlerocket-update-operator/pkg/logging"
+	"github.com/bottlerocket-os/bottlerocket-update-operator/pkg/platform"
+
+	"github.com/pkg/errors"
```

Comment:
Earlier code wasn't as careful to arrange imports all around (my bad). Now, as we go, let's try to keep these ordered consistently when working with imports:

```suggestion
	"github.com/Masterminds/semver"
	"github.com/pkg/errors"
	
	"github.com/bottlerocket-os/bottlerocket-update-operator/pkg/logging"
	"github.com/bottlerocket-os/bottlerocket-update-operator/pkg/platform"
```